packages/tracing-internal/src/browser/request.ts

```diff
@@ -195,6 +197,7 @@ export function fetchCallback(
       data: {
         ...handlerData.fetchData,
         type: 'fetch',
+        ...(contentLength ? { 'Encoded Body Size': contentLength } : {}),
```

Comment:
h: I actually think we should try and deprecate and move away from `Encoded Body Size` - having a key with capitals + spaces seems like an anti-pattern, can we try to rename this and update the sentry code accordingly?

Reply:
@AbhiPrasad Is there a convention for params like this? camel/snake case? I'm taking over the rollout of this perf issue so I'll make the changes

Reply:
https://develop.sentry.dev/sdk/performance/span-data-conventions/ is the list.

> Keys on the data field should be lower-case and use underscores instead of camel-case. There are some exceptions to this, but these exist because of backwards compatibility.

`.` separators are also used.

Reply:
Hmm, I made a change to switch to `encoded_body_size` here, but I noticed that other perf issues also depend on values in this data object such as "Transfer Size", "Encoded Body Size", and "Decoded Body Size". Do you know if these keys would count as exceptions to the rule? It feels like renaming here would be inconsistent

Reply:
Why did we revert this key change? No perf detector should depend on it atm - they should only be looking at resource spans right? I feel strongly about making sure we follow the schema here.

Reply:
Isn't it documented here? https://develop.sentry.dev/sdk/performance/span-data-conventions/#browser

And there are some detectors using it, i.e. [the uncompressed asset detector](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/performance_issues/detectors/uncompressed_asset_detector.py#L40) and the [render blocking asset detector](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/performance_issues/detectors/render_blocking_asset_span_detector.py#L103)

Reply:
it is documented, but it was actually added first in the code, and later in conventions.

It shouldn't be that hard to update the detector logic right? I understand this is more work than maybe you've signed up for 😅 - but this is the cleaner solution long term - and I'd like to keep the schema consistent.

Reply:
> No perf detector should depend on it atm

I was mistaken here - I thought the perf detectors would restrict to resource spans since that only sends data atm, but from reading source code I guess `http` works as well.

Reply:
also @AbhiPrasad I don't think we ever changed to snake_case we just talked about it right? it's bothered me as well but I'm not sure we made plans to change encoded/decoded to snake yet.. or did we and I forget?

Reply:
otel now uses snake case so I think we try and match that (for ex: https://github.com/open-telemetry/opentelemetry-specification/pull/3287/files).

Reply:
Also, should it be `http.response_content_length`? That seems to be what [OTel is suggesting](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#common-attributes)

Reply:
yeah @gggritso that makes sense to me. @narsaynorath could we first open a PR on develop to https://github.com/getsentry/develop to establish the naming, and then update both sentry/sdks?